providers/implementations/ciphers/cipher_aes_xts.c

```diff
@@ -20,11 +20,11 @@
 #define AES_XTS_IV_BITS 128
 #define AES_XTS_BLOCK_BITS 8
 
-#ifdef FIPS_MODE
-static const int allow_insecure_decrypt = 0;
-#else
-static const int allow_insecure_decrypt = 1;
-#endif /* FIPS_MODE */
+/*
+ * Available in cipher_fips.c, and compiled with different values depending
+ * on we're in the FIPS module or not.
+ */
+extern const int allow_insecure_decrypt;
```

Comment:
Why is this needed here. Don't we pick this up from cipher_aes_xts.h?

Reply:
Oh!  I thought I had cleaned that away...  Will do